DigitalLearningSolutions.Data/DataServices/UserDataService/UserDataService.cs

```diff
@@ -486,5 +488,16 @@ END CATCH
                 throw new InactivateUserUpdateException(message);
             }
         }
+
+        public void ActivateUser(int userId)

```

Comment:
We should probably have a test for this. I think Sarathlal included one for the Inactivate method which might be useful as a reference.

Reply:
this is fixed now